doc/rados/operations/pools.rst

```diff
@@ -378,45 +381,41 @@ You may set values for the following keys:
 
 .. describe:: pg_num
 
-   The effective number of placement groups to use when calculating
+   Sets the effective number of PGs to use when calculating
    data placement.
 
    :Type: Integer
-   :Valid Range: Superior to ``pg_num`` current value.
+   :Valid Range: Greater than the current value of ``pg_num``.
 
 .. _pgp_num:
 
 .. describe:: pgp_num
 
-   The effective number of placement groups for placement to use
-   when calculating data placement.
+   Sets the effective number of PGs to use when calculating data placement.
 
    :Type: Integer
-   :Valid Range: Equal to or less than ``pg_num``.
+   :Valid Range: Less than or equal to the current value of ``pg_num``.
 
 .. _crush_rule:
 
 .. describe:: crush_rule
 
-   The rule to use for mapping object placement in the cluster.
+   Sets the rule that Ceph uses to map object placement in the cluster.
 
    :Type: String
 
 .. _allow_ec_overwrites:
 
 .. describe:: allow_ec_overwrites
 
-
-   Whether writes to an erasure coded pool can update part
-   of an object, so cephfs and rbd can use it. See
-   `Erasure Coding with Overwrites`_ for more details.
+   Determines whether writes to an erasure-coded pool are allowed
+   to update part of an object so that CephFS and RBS can use the object.
+   For more details, see `Erasure Coding with Overwrites`_.
```

Comment:
Determines whether writes to an erasure-coded pool may update only part of a RADOS object.  This allows CephFS and RBD to use the pool for data.

Reply:
I'm going to accept this, but I think that the phrase "to use the pool for data" is vague. This is a danger of using for-phrases. So: what exactly does it mean to use a pool for data? (I'm probably going to merge this as it is, but I will have no compunction to raise a future pull request to make this sentence better.)

Reply:
As opposed to metadata.  I almost wrote “user data” or “payload data”

Reply:
@zdover still need to s/RBS/RBD/

Reply:
Oh, good Lord. You are correct.

Reply:
Good looking out.

Reply:
See below for refinement